firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+URLBarDelegate.swift

```diff
@@ -35,6 +35,10 @@ extension BrowserViewController: URLBarDelegate {
         updateFindInPageVisibility(visible: false)
 
         guard !isTabTrayRefactorEnabled else {
+            if let tab = tabManager.selectedTab {
+                screenshotHelper.takeScreenshot(tab)
+            }
+
             navigationHandler?.showTabTray(selectedPanel: focusedSegment ?? .tabs)
             return
         }
```

Comment:
I don't really like putting this in this guard, but also didn't like passing the `screenshotHelper` to `showTabTray` so I'm open to suggestions here.